core/src/components/input/test/shape/input.e2e.ts

```diff
@@ -109,3 +109,60 @@ configs({ modes: ['ionic-md', 'md'] }).forEach(({ title, screenshot, config }) =
     });
   });
 });
+
+// Rectangular shape is only available in ionic theme
+// TODO(FW-6098): Add test for rectangular shape in md
+// by combining these tests with the above tests
```

Comment:
Nit pick: Move the comment above of line 118. I also have a similar test for soft. It would be easier to read the comments if they're near their addressed test vs all at the top.

Reply:
Moved in https://github.com/ionic-team/ionic-framework/pull/29476/commits/329585d9fb9a0c865bd3de38fe1b6b492fd970ee